lib/src/kbasesearchengine/main/SearchMethods.java

```diff
@@ -124,13 +124,17 @@ private static Integer toInteger(Long value) {
     private kbasesearchengine.search.AccessFilter toSearch(AccessFilter af, String user)
```

Comment:
There's a search types, search objects, and get objects that have auth changes in the server, but only 2 methods that have changes here. Is there a method that's missing necessary changes?

Reply:
looks like getObjects already have takes care of the case in which the user is null

Reply:
wait nvm thats umas changes